apps/02RockPaperScissors.js

```diff
@@ -9,15 +9,40 @@ var rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
 
   // Write code here
+  if (hand1 === hand2) {
+    return "It's a tie!";
+  } else if (hand1 === "rock") {
 
+    if (hand2 === "scissors") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+  } else if (hand1 === "paper") {
+
+    if (hand2 === "rock") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
+    }
+  } else if (hand1 === "scissors") {
+
+    if (hand2 === "rock") {
+      return "Hand one wins!";
+    } else {
+      return "Hand two wins!";
```

Comment:
adding other options for entry i think could also make this  a more exciting game, or make it a 2-out-of-three or 3-out-of-five sort of competition would give an opportunity for more complex code.